### ISSUE TITLE
Review potential name changes

### ISSUE THREAD

**Issue by kuwv** [python-proman/versioning#29]
*Overview*

Proman can be potentially misconstrued to be other than a portmantau of project management.

*Acceptance*

Find an alternative project name or group namespace that is easier to remember and potentailly a better fit.

**Comment by kuwv** [comment]
@mjbrisebois I'm trying to rename this project I main. I see you have a project you no longer maintain. Would you be ok with transferring the PyPI project so that I can simplify this project name?

**Comment by mjbrisebois** [comment]
Sure.  What is your pypi username?

**Comment by kuwv** [comment]
> Sure.  What is your pypi username?

Mu username is the same as my GitHub user "kuwv".

**Comment by kuwv** [comment]
@mjbrisebois friendly reminder, not sure if you were notified of my previous comment or not.

**Comment by mjbrisebois** [comment]
Yes, I was not getting notifications for this so thanks for the reminder.  I added you as an owner.  Feel free to take it over and remove me.
